core/src/main/java/io/questdb/network/IOContextFactoryImpl.java

```diff
@@ -52,7 +52,7 @@ public void done(C context) {
         if (closed) {
             Misc.free(context);
         } else {
-            context.of(-1, null);
+            context.of(null, null);
```

Comment:
it would be "clearer" to introduce `clear()` method to do this

Reply:
Done in 5ee92af47fde755e6e35cfd805020f8dc848e34f